core/extensions/COMServerDemo/ReadMe.md

```diff
@@ -31,7 +31,7 @@ Program should output an estimated value of &#960;.
 
 ### RegFree COM ###
```

Comment:
Perhaps in the Note (on line 30) it would also be useful to include command-line syntax for unregistering.